xga/generate/ciao/run.py

```diff
@@ -0,0 +1,220 @@
+from functools import wraps
```

Comment:
Need to add the standard file header:

\#  This code is a part of X-ray: Generate and Analyse (XGA), a module designed for the XMM Cluster Survey (XCS).
\#  Last modified by David J Turner (turne540@msu.edu) 21/11/2024, 10:57. Copyright (c) The Contributors

Though obviously modify to your name and details

We'll have to see if vscode can do that automatically, PyCharm adds it to every new file I make in the project and auto updates the time and date whenever I commit something

Reply:
Should I use my name and email here? I saw all other headers are having your names.